OracleWebLogic/samples/12213-domain-home-in-image/README.md

```diff
@@ -73,17 +76,19 @@ To start the containerized Administration Server, run:
           -v <HOST DIRECTORY TO PROPERTIES FILE>/properties/docker-run:/u01/oracle/properties \
           12213-domain-home-in-image
 
+If both the JAVA_OPTIONS environment variable and the JAVA_OPTIONS in the docker-run/security.properties file are configured, the latter gets appended to the former. If SSL is enabled, pass "-e JAVA_OPTIONS=-Dweblogic.security.SSL.ignoreHostnameVerification=true" to the "docker run ..." command that starts a managed server because the Demo identity certificate got generated with a host name different from the admin host above when the domain was created.
```

Comment:
Instead of "command that starts a managed server  because the Demo identity certificate got generated with a host name different from the admin host above when the domain was created." it reads better "command that starts a managed server.  This is necessary because the Demo identity certificate gets generated with a host name different from the admin server host when the domain is created."

Reply:
These changes should also be made to the sample domain-home-in-image-wdt

Reply:
@mriccell , I've updated the README.md as suggested.  I also made the same changes to enable SSL in the WDT sample.  Please take a look. Thanks!